ricardo-leite.txt

```diff
@@ -1 +1,138 @@
-Hello World!
\ No newline at end of file
+Hello World!
+
+1. What is Semantic HTML? - 
+	Semantic HTML is what gives meaning and structure for the HTML code.
+
+    2. What is HTML used for? 
+	HTML is used to markup text to be displayed on the web.
+
+    3. What is an attribute and where do we put it? 
+	An attribute are words used inside the tags to control the behavior of the element later on with css. 	
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+	The h1 tag is an element to be used as a header for the text. It is best to use one h1 tag per page.
+
+    5. Name two tags that have required attributes
+	class and id
+
+    6. What do we put in the head of our HTML document? 
+	<head>
+
+    7. What is an id? 
+	A selector
+
+    8. What elements can I add an id to? 
+	Any element, as long as it's in only one
+
+    9. How many times can I use the same id on a page? 
+	Once
+
+    10. What is a class? 
+	A selector
+
+    11. What elements can I add a class to? 
+	Any HTML element
+
+    12. How many times can I use the same class on a page? 
+	Multiple times
+
+    13. How do I get my link to open in a new tab?
+	<a href="link"></a>
+
+    14. What is the alt attribute used for? 
+	It is used to specify alternative text
+
+    15. How do I reference an id?
+	With the #
+
+    16. What is the difference between a section and a div
+	Section means that the content inside is grouped, while div tags define a division
+
+    17. What is CSS used for? 
+	Styling
+
+    18. How to we select an element? Example - every h2 on the page
+	h2{}
+
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one
+	A class can be used multiple times in the page to identify multiple elements, while an ID is used to identify one element. 
+	EX: #1 {
+	color: black;
+	}
+	EX: .2 {
+	color: black
+	}
+
+    20. How do we select classes in CSS?
+	With the period " . "
+
+    21. How do we select a p element with a single class of “human””?
+	With just "p"
```

Comment:
Create two p elements and add a human class to one and see if you answer is correct.